build fail looks real
```
======================================================================
FAIL: test_report_module (corehq.apps.app_manager.tests.test_suite.SuiteTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/test_suite.py", line 661, in test_report_module
    "./detail[@id='reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.summary']",
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/util.py", line 20, in assertXmlPartialEqual
    self.assertXmlEqual(expected, actual, normalize=False)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/util.py", line 26, in assertXmlEqual
    _check_shared(expected, actual, LXMLOutputChecker(), "xml")
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/app_manager/tests/util.py", line 117, in _check_shared
    raise AssertionError(message)
AssertionError: XML mismatch
--- want.xml
+++ got.xml
@@ -66,7 +66,7 @@
     <detail id="reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.data" nodeset="rows/row">
       <title>
         <text>
-          <locale id="cchq.reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.name"/>
+          <locale id="cchq.report_data_table"/>
         </text>
       </title>
       <field>
```

--- a/corehq/apps/app_manager/tests/data/suite/reports_module_data_detail.xml
+++ b/corehq/apps/app_manager/tests/data/suite/reports_module_data_detail.xml
@@ -2,7 +2,7 @@
   <detail id="reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.data" nodeset="rows/row">
     <title>
       <text>
-        <locale id="cchq.reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.name"/>
+        <locale id="cchq.report_data_table"/>
       </text>
     </title>
     <field>

--- a/corehq/apps/app_manager/tests/data/suite/reports_module_summary_detail.xml
+++ b/corehq/apps/app_manager/tests/data/suite/reports_module_summary_detail.xml
@@ -66,7 +66,7 @@
     <detail id="reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.data" nodeset="rows/row">
       <title>
         <text>
-          <locale id="cchq.reports.ip1bjs8xtaejnhfrbzj2r6v1fi6hia4i.name"/>
+          <locale id="cchq.report_data_table"/>
         </text>
       </title>
       <field>

